external-crates/move/crates/move-binary-format/src/file_format_common.rs

```diff
@@ -19,6 +19,54 @@ use std::{
     mem::size_of,
 };
 
+// Static assertions about the encoding of the flavor into the version of the binary format.
+const _: () = {
```

Comment:
Is it worth naming this for easier debugging if the assertions ever fail?

Reply:
Probably not, but I'll add some more comments!